dotcom-rendering/src/web/components/SignInGate/gates/main-variant.tsx

```diff
@@ -15,19 +15,63 @@ const SignInGateMain = React.lazy(() => {
 	});
 });
 
+const SignInGateMainCheckoutComplete = React.lazy(() => {
+	const { start, end } = initPerf('SignInGateMainCheckoutComplete');
+	start();
+	return import(
+		/* webpackChunkName: "SignInGateMain" */ '../gateDesigns/SignInGateMainCheckoutComplete'
+	).then((module) => {
+		end();
+		return { default: module.SignInGateMainCheckoutComplete };
+	});
+});
+
+/**
+ * GATE PERSONALISATION:
+ *
+ * If the GU_CO_COMPLETE cookie is present, personalise the sign in gate based on the userType
+ * and product information in the cookie value.
+ * AB tracking will be registered as part of the main gate AB test, but the component Id string
+ * in Ophan ComponentEventTracking will have `_personalised_${userType}_${product}` appended.
+ * This occurs in the SignInGateSelector.importable.tsx file
```

Comment:
It might be good to add a comment here about the Switch and where it's being read?